db/post_migrate/20200629232159_rename_path_whitelist_to_allowed_paths.rb

```diff
@@ -2,6 +2,6 @@
 
 class RenamePathWhitelistToAllowedPaths < ActiveRecord::Migration[6.0]
   def change
-    rename_column :embeddable_hosts, :path_whitelist, :allowed_paths
+    rename_column :embeddable_hosts, :path_whitelist, :allowed_paths unless column_exists?(:embeddable_hosts, :allowed_paths)
```

Comment:
I think this migration can be removed completely?

Reply:
It can be done, but... some systems are already migration so we will have a number in `schema_migrations` without corresponding file. In theory, it is fine, but should we keep it as a reference?

Reply:
I think there isn't a good reason to keep it as a reference since it adds an overhead to all new Discourse instance.